manifests/init.pp

```diff
@@ -27,18 +27,26 @@
 #
 # @param manage_properties
 #   Boolean specifying whether or not to manage the catalina.properties file. Defaults to true.
+#
+# [*copy_from_home*]
```

Comment:
Removed from init.pp altogether.  Do the inline docs for instance.pp need to be updated as well?